frontend/sw360-portlet/src/main/webapp/html/utils/ajax/userListAjax.jsp

```diff
@@ -29,17 +30,28 @@
                 <core_rt:if test="${how ==false}">
                         type="radio"
                 </core_rt:if>
-                        name="id" value="<sw360:out value="${entry.email}, ${entry.fullname}"/>">
+                        name="id" value="<sw360:out value="${entry.givenname},${entry.lastname},${entry.email},${entry.department},${entry.fullname}"/>">
             </td>
-            <td><sw360:out value="${entry.fullname}"/></td>
+            <td><sw360:out value="${entry.givenname}"/></td>
+            <td><sw360:out value="${entry.lastname}"/></td>
+            <td><sw360:DisplayUserEmail email="${entry.email}" bare="true"/></td>
+            <td><sw360:out value="${entry.department}"/></td>
         </tr>
     </core_rt:forEach>
 </core_rt:if>
 <core_rt:if test="${userList.size() == 0}">
     <tr>
-        <td colspan="2">
+        <td colspan="5">
             No user found with your search.
         </td>
     </tr>
 
 </core_rt:if>
+
+<script>
```

Comment:
I agree that this should be changed at some point. But I would suggest to do this when dealing with the duplicate entries mentioned in your review comment (e.g. if the ajax-query would be changed to use JSON this might be easier).

Reply:
Okay :-)